### ISSUE TITLE
Adding Deaths to the Parser

### ISSUE THREAD

**Issue by captawesom3** [nettyneets/apex-n#4]
Is it possible to add how many times a player has died during the encounter onto the parser.

**Comment by nyliya** [comment]
Yes, that's completely possible. Would you suggest that to be displayed next to every player, or just yourself? (eg the person parsing)

**Comment by captawesom3** [comment]
For me I would prefer next to every player.
